src/Cardano/Shell/Types.hs

```diff
@@ -17,7 +17,7 @@ import           Control.Concurrent.Classy (MonadConc)
 
 import           Cardano.Shell.Constants.Types (CardanoConfiguration (..))
 
-import qualified System.Metrics as Ekg
+-- import qualified System.Metrics as Ekg
```

Comment:
Do we need these comments? EKG is not really an issue, the stack traces just indicate something went wrong, not that it's crashing the whole application.

Reply:
Commented out code should never be merged to master.